src/app/views/query-runner/query-input/auto-complete/AutoComplete.tsx

```diff
@@ -1,6 +1,7 @@
-import { getId, getTheme, Icon, ITextField, KeyCodes, Spinner, TextField, TooltipHost } from 'office-ui-fabric-react';
+import { getId, getTheme, Icon, ITextField, KeyCodes, Spinner, TooltipHost } from 'office-ui-fabric-react';
 import { ITooltipHostStyles } from 'office-ui-fabric-react/lib/components/Tooltip/TooltipHost.types';
 import React, { Component } from 'react';
+import { TextField } from '@fluentui/react/lib/TextField';
```

Comment:
TextField can be imported from within `office-ui-fabric-react` instead, since the move to fluent UI is pending